ranged_weapon.cpp

```diff
@@ -57,6 +59,11 @@ void RangedWeapon::fire(WCreature c, Vec2 dir) const {
       pos.globalMessage("the " + projectileName + " hits the " + pos.getName());
       break;
     }
+    if (distance >= MAX_RANGED_DISTANCE)
+    {
```

Comment:
Please put the { in the line above, similar to like all other code is formatted.